src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -388,6 +394,22 @@ public function newLine($count = 1)
     }
 
     /**
+     * Sets the input stream used by the SymfonyQuestionHelper.
+     *
+     * @param resource $stream
+     *
+     * @throws InvalidArgumentException If the given stream is not a resource
+     */
+    public function setInputStream($stream)
+    {
+        if (!is_resource($stream)) {
+            throw new InvalidArgumentException(sprintf('Input stream must be a valid resource, %s given', gettype($stream)));
```

Comment:
I hesitated but It seems that types are unquoted when the method is not expecting a specific class instance so it should be ok. See the [DI ParameterBag](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php#L217) for instance.